src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -226,6 +226,11 @@ private static OpenType GetOpenType(Type t)
                 }
             }
 
+            if (typeDest == typeof(JArray) && typeof(IEnumerable).IsAssignableFrom(typeSource))
```

Comment:
Ah, you're right.  Way too locked into the POCO mindset.

---
In reply to: [103005608](https://github.com/Azure/azure-webjobs-sdk/pull/1037#discussion_r103005608) [](ancestors = 103005608)